src/core/Akka/Actor/Internals/ActorSystemImpl.cs

```diff
@@ -53,9 +53,9 @@ public ActorSystemImpl(string name, Config config)
 
             _name = name;            
             ConfigureSettings(config);
+            ConfigureScheduler();
             ConfigureEventStream();
             ConfigureProvider();
-            ConfigureScheduler();
```

Comment:
this is the problem, the scheduler needs to be configured after the provider so the dedicated thread scheduler can access the provider